lib/rutabaga/feature.rb

```diff
@@ -3,67 +3,22 @@
 
 module Rutabaga
   module Feature
-    def feature(feature_file = find_feature)
-
+    def feature(feature_file = nil)
+      RSpec.deprecate(
+        "Calling `feature` from an `it` block",
+        :message => "Calling `feature` from an `it` block " \
+                    "is deprecated.\nIt should now be called directly in the " \
+                    "`describe` block."
+      )
+
+      feature_file = Util.find_feature(feature_file || RSpec.current_example.description)
       example_group_class = self.class
 
       # Hack turnip into the rspec only when needed
       example_group_class.send(:include, Turnip::RSpec::Execute)
```

Comment:
i would suggest to look for the feature tag, now you basically allowing any rspec to use turnip steps.

but i guess you cant use the (type)feature tag, otherwise the rspec rails helpers wont work... so you will need to think of a better way to handle this i would imagine.